### DISCUSSION TITLE
Automatically hide the anchors when the mouse is hidden

### DISCUSSION THREAD

**Discussion by ltxman** [WindowTop/WindowTop-App#223] [Ideas]
How to automatically hide the anchor when playing a video in full screen?
Is there a function of automatically hiding anchors in full screen? It is inconvenient to manually close the anchor every time the video is played in full screen.

**Comment by gileli121** [comment]
@ltxman 
Currently, such behavior is not implemented yet.
I will probably implement it one day but to make it happen faster, need more votes here.

For now, you can configure a hotkey to enable/disable the anchors.
At least this way maybe it will be easier 

To set hotkey, 

1. Open the settings window:
![image](https://user-images.githubusercontent.com/17680514/167268552-76aa13d6-ddcf-4900-9eba-61bb3d515ed0.png)

2. Go the the Anchors tab:
![image](https://user-images.githubusercontent.com/17680514/167268581-51599cd6-2323-4718-8e16-a4722ef36a3b.png)

3. Scroll down to the hot-key section:
![image](https://user-images.githubusercontent.com/17680514/167268593-7d6f33c3-46f1-46d5-b518-ed2fa0774743.png)

Click inside the input box so it will be highlighted in a blue border and type the key combination.

Now to enable the anchors, press the hotkey. To disable the anchors, press the hotkey again.

**Reply by ltxman** [reply]
OK，thank you.

**Comment by gileli121** [comment]
@ltxman 
What do you think about the idea to hide the anchors when the mouse is hidden also?
Any fullscreen player will hide the mouse.
And also the mouse is hiding in some apps while you typing.

In any case, when the mouse is hidden you don't need the anchors anyway.
For me, it makes sense to hide the anchors while the mouse is hidden.

**Comment by ltxman** [comment]
I think this idea is great and ingenious! And I look forward to the realization of this function. Of course, you may need more votes.



---- Replied Message ----
| From | Gil ***@***.***> |
| Date | 06/17/2022 22:57 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [BiGilSoft/WindowTop] How to automatically hide the anchor when playing a video in full screen? (Discussion #223) |

@ltxman
What do you think about the idea to hide the anchors when the mouse is hidden also?
Any fullscreen player will hide the mouse.
And also the mouse is hiding in some apps while you typing.

In any case, when the mouse is hidden you don't need the anchors anyway.
For me, it makes sense to hide the anchors while the mouse is hidden.

**Reply by gileli121** [reply]
@ltxman 
Great
I changed the title to "Automatically hide the anchors when the mouse is hidden"
And I also vote for this feature.

This is also easy to implement. When something is easy to implement, it may help in addition to the votes
